### DISCUSSION TITLE
can you help me about fleet api?

### DISCUSSION THREAD

**Discussion by rhatami7495** [tweepy/tweepy#1590] [Questions] [answered]

**Comment by Harmon758** [comment, marked as answer]
If you're referring to Twitter Fleets, that's not something Twitter's API supports.
See:
https://twittercommunity.com/t/will-twitter-api-support-fleets/145710
https://twittercommunity.com/t/failed-to-request-fleets/146278
https://twitterdevfeedback.uservoice.com/forums/930250-twitter-api/suggestions/41994790-fleets
